lab/src/Microsoft.ServiceModel.Syndication/src/SyndicationElementExtension.cs

```diff
@@ -23,20 +23,20 @@ public class SyndicationElementExtension
         private string _outerName;
         private string _outerNamespace;
 
-        public SyndicationElementExtension(XmlReader xmlReader)
+        public SyndicationElementExtension(XmlReader reader)
         {
```

Comment:
This is doing a lot of work in the constructor synchronously. You don't need to change this for this CR but this needs to be refactored so it's done by an Async method call.